geodepy/fileio.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
```

Comment:
Should this just be part of the inputoutput module? Or should we merge the inputoutput module into fileio?

Reply:
I don't know. I wanted to keep it separate for now. I need to have a look at what's in inputoutput. It depends on pandas and I'm wary of making too many dependencies. I think an external package has to offer a significant benefit to warrant inclusion.

Reply:
Ok that's fine, something for us to look at and possibly flag as an issue.
Merge away!